tests/linker/ios/link sdk/CryptoTest.cs

```diff
@@ -182,7 +182,12 @@ public void Chain ()
 			Assert.False (chain.Build (cert), "Offline");
 
 			chain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
+#if __MACOS__
+			// Not sure if this is expected on macOS, or if it's a bug somewhere in our code.
+			Assert.False (chain.Build (cert), "NoCheck");
```

Comment:
does that happen on both mono and dotnet runtimes ?

Reply:
Only .NET.

I'll create a test case and file an issue in dotnet/runtime.

Reply:
That said, the other .NET platforms hit this case:

https://github.com/xamarin/xamarin-macios/blob/213dbd2c1d7e835a062313dd94d01f28e24d0052/tests/linker/ios/link%20sdk/CryptoTest.cs#L162

Reply:
https://github.com/dotnet/runtime/issues/57630